docs/elements.md

```diff
@@ -1782,8 +1782,8 @@ Along with the sample usage of the element, we include a sample template file. T
   <div class="card-header" style="cursor: pointer">
     <div
       class="card-title d-flex justify-content-between"
-      data-toggle="collapse"
-      data-target="#collapse-{{uuid}}"
+      data-bs-toggle="collapse"
+      data-bs-target="#collapse-{{uuid}}"
```

Comment:
Same note as in the example course, this isn't accessible, we should either improve this or find a different example.

Reply:
How hard is it to make changes to the example? I think adding the `aria-*` attributes seems straightforward (and in fact, doing this through the template element would be good for demonstrating ways for instructors to make content that is accessible), but I'm not sure how to with this using a keyboard.

Reply:
This is now tracked in https://github.com/PrairieLearn/PrairieLearn/issues/11444.